workshops/views.py

```diff
@@ -1794,6 +1796,9 @@ def workshop_staff(request):
                 people = people.filter(q, task__role__name='learner') \
                                .exclude(badges__in=instructor_badges)
 
+            if data['languages']:
+                people  = people.filter(languages__in=data['languages'])
+
```

Comment:
I think there are some tests for `workshop_staff` searching, please extend them to cover this addition too.